app/views/unattended/provisioning_templates/registration/global_registration.erb

```diff
@@ -106,7 +106,8 @@ fi
 <% end -%>
 
 register_host() {
-  curl --silent --show-error --cacert $SSL_CA_CERT --request POST <%= @registration_url %> \
+    CURL_OUT=$(mktemp)
+    HTTP_RETURN_CODE=$(curl --silent --output $CURL_OUT --write-out "%{http_code}"  --cacert $SSL_CA_CERT --request POST <%= @registration_url %> \
```

Comment:
Any reason not to use `--fail`? It's available on EL7. Perhaps not on EL6?

Reply:
I didn't try it because: `Fail silently (no output at all)` and I wanted to show the error output.